src/System.Configuration.ConfigurationManager/tests/System/Configuration/UrlPathTests.cs

```diff
@@ -25,14 +25,18 @@ public void GetDirectoryOrRootName_NotDirectoryOrRoot()
         }
 
         [Fact]
-        public void GetDirectoryOrRootName_FileExists()
+        public void GetDirectoryOrRootName_GettingDIrectoryFromAFilePath()
         {
+
             string exePath = AppDomain.CurrentDomain.BaseDirectory;
+            //Remove the trailing slash.  Different OS's use a different slash.
```

Comment:
Please put a space immediately after // 
I know we aren't yet consistent elsewhere.